explorer/interpreter/interpreter.cpp

```diff
@@ -788,6 +788,10 @@ auto Interpreter::CallFunction(const CallExpression& call,
           alt.alt_name(), alt.choice_name(), arg));
     }
     case Value::Kind::FunctionValue: {
+      if (phase() == Phase::CompileTime) {
+        return ProgramError(call.source_loc())
+               << " function call not allowed in this context";
```

Comment:
```suggestion
               << "Function would be called at compile-time";
```

Prefix space shouldn't be there (note how it results in a double space in error messages)

And I think we can actually be more specific here; "in this context" seems ambiguous to me.